File: Day - 2/question1.c
#include <stdio.h>
int main(){
    /*Write a program to Find sum of digits of a 
number. */
int n , d , s = 0;
printf("enter the number n =");
scanf("%d", &n);
while(n>0){
    d = n%10;
    s = s + d;
    n = n/10;
}
printf("the sum of digit is %d", s);
return 0;
}

File: Day - 15/Question 1.py
#Write a program to Reverse array. 

n = int(input("enter the number of elements = "))

# Input array elements
array = []
for i in range(n):
    element = int(input(f"enter element {i+1} = "))
    array.append(element)

# Reverse the array
array.reverse()

# Display reversed array
print("Reversed array:", array)
